File: php/function/palindrome_numbers.php
<?php
// Program to print all palindrome numbers from 1000 to 1500

function isPalindrome($number)
{
    $original = $number;
    $reverse = 0;

    while ($number > 0) {
        $digit = $number % 10;
        $reverse = $reverse * 10 + $digit;
        $number = (int) ($number / 10);
    }

    return $original == $reverse;
}

echo "Palindrome numbers from 1000 to 1500 are:\n";

for ($i = 1000; $i <= 1500; $i++) {
    if (isPalindrome($i)) {
        echo $i . "\n";
    }
}
?>

File: java/Array/PalindromeNumbers.java
class PalindromeNumbers {
    public static void main(String[] args) {
        int[] numbers = {121, 1212, 3132, 1331, 135, 17971, 5343, 2345};

        System.out.println("Palindrome numbers:");

        for (int originalNumber : numbers) {
            int number = originalNumber;
            int reversedNumber = 0;

            // Reverse the number
            while (number > 0) {
                int digit = number % 10;
                reversedNumber = reversedNumber * 10 + digit;
                number /= 10;
            }

            // Check if the original number is equal to its reversed version (Palindrome check)
            if (originalNumber == reversedNumber) {
                System.out.println(originalNumber + " is a palindrome number.");
            }
        }
    }
}
